server/chat-plugins/room-events.ts

```diff
@@ -19,9 +32,19 @@ function formatEvent(event: RoomEvent, showAliases?: boolean) {
 	if (!isNaN(timeRemaining)) {
 		explanation = `This event will start in: ${Chat.toDurationString(timeRemaining, {precision: 2})}`;
 	}
+
+	const eventID = toID(event.eventName);
+	const aliases = getAllAliases(room).filter(
+		alias => (room.settings.events![alias] as RoomEventAlias).eventID === eventID
+	);
```

Comment:
This suggests to me that `getAllAliases` isn't the right level of abstraction. You might perhaps want to pass an `eventID` parameter, so you could do `getAliases(room)` but also `getAliases(room, eventID)` to only get aliases for a specific event.